doc/tools/opensc-explorer.1.xml

```diff
@@ -104,13 +113,32 @@
 			interactive prompt or in script files passed via the command line parameter
 			<replaceable class="parameter">SCRIPT</replaceable>.
 			<variablelist>
+				<varlistentry>
+					<term>
+						<command>#</command>
+						<arg choice="plain" rep="repeat"><replaceable></replaceable></arg>
```

Comment:
wouldn't this make more sense with `<arg choice="plain" rep="repeat"><replaceable>comment</replaceable></arg>`?

Reply:
This piece of DocBook yields   `# ...` in the manual page.
I do not really care whether it stays as it is or becomes `# comment...`
If you explicitly want `comment` included, please tell and I amend the PR.